docs/tutorials/timeseries/forecasting-indepth.md

```diff
@@ -0,0 +1,248 @@
+# Forecasting Time Series - In Depth
+:label:`sec_forecastingadvanced`
+
+This tutorial provides an in-depth overview of the time series forecasting capabilities in AutoGluon.
+
+- What is probabilistic time series forecasting?
+- What forecasting models are available in AutoGluon?
+- How does AutoGluon evaluate performance of time series models?
+- What functionality does `TimeSeriesPredictor` offer?
+    - Choosing the presets
+    - Manually specifying what models should be trained
+    - Hyperparameter tuning
+    - Forecasting irregularly-sampled time series
+
+This tutorial assumes that you are familiar with the contents of the [basic tutorial](forecasting-quickstart.md).
+
+## What is probabilistic time series forecasting?
+A time series is a sequence of measurement made at regular intervals.
+The main objective of time series forecasting is to predict the future values of a time series given the past observations.
+
+A typical example of this task is demand forecasting.
+We can represent the number of daily purchases of a certain product as a time series.
+The goal in this case can be to predict the demand for each of the next 14 days given the historical purchase data.
+
+In AutoGluon, the `prediction_length` argument of the `TimeSeriesPredictor`
+determines the length of the forecast horizon.
+
+![ ](https://autogluon-timeseries-datasets.s3.us-west-2.amazonaws.com/public/figures/forecasting-indepth1.png)
```

Comment:
For some reason in the [automatically build documentation](http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-2146/df911da/tutorials/timeseries/forecasting-indepth.html) this link is changed and the `:width:` flag isn't rendered correctly. 

I don't have neither of these problems when building locally, any ideas what might be causing this?

Reply:
Probably d2lbook messing with the rendering. This is something we are getting rid of in the new website. You don't have to worry about it for now.